src/crewai/tools/tool_usage.py

```diff
@@ -103,6 +104,19 @@ def use(
             if self.agent.verbose:
                 self._printer.print(content=f"\n\n{error}\n", color="red")
             return error
+
+        if isinstance(tool, CrewStructuredTool) and tool.name == 'Add image to content':
```

Comment:
Still worried about translations, but actually more with having logic tied to tool name. I mean, what if someone sets up a tool with this very name?

Reply:
Missed the translation here?